get-started/index.md

```diff
@@ -62,7 +65,302 @@ the specifics of how to deploy your Cryostat instance. Continue to [Setup](#setu
+  - How to use the Cryostat Agent to do application discovery and expose data over HTTP.
+  - How to use Kubernetes `Service` configurations for discovery and JMX to expose data.
+
+There is a third, hybrid approach: **using the Cryostat Agent to do application discovery, and JMX to expose data**. This may be
+useful since the Agent HTTP data model is readonly, whereas JMX is read-write. This means that using JMX to communicate between Cryostat and your applications
+allows for more dynamic flexibility, for example, the ability to start and stop Flight Recordings on demand. Using the Cryostat Agent for application discovery
+is also more flexible than depending on `Services` with specially-named or specially-numbered ports. 
+
+For more context about these concepts, please review
+the previous two sections on [using the Cryostat Agent](#using-the-cryostat-agent) and [using JMX](#using-jmx).
+
+Add dependency configuration to `pom.xml`:
+```xml
+<project>
+  ...
+  <repositories>
+    <repository>
+      <id>github</id>
+      <url>https://maven.pkg.github.com/cryostatio/cryostat-agent</url>
+    </repository>
+  </repositories>
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.3.0</version>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>io.cryostat</groupId>
+                  <artifactId>cryostat-agent</artifactId>
+                  <version>0.2.0</version>
+                </artifactItem>
+              </artifactItems>
+              <stripVersion>true</stripVersion>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+    ...
+  </build>
+  ...
+</project>
+```
+
+Modify the `application Deployment`:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+...
+spec:
+  ...
+  template:
+    ...
+    spec:
+      containers:
+        - name: sample-app
+          image: docker.io/myorg/myapp:latest
+          env:
+            - name: CRYOSTAT_AGENT_APP_NAME
+              value: "myapp"
+            - name: CRYOSTAT_AGENT_BASEURI
+              value: "http://cryostat.mynamespace.mycluster.svc:8181"
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.PodIP
+            - name: CRYOSTAT_AGENT_CALLBACK
+              value: "http://$(POD_IP):9977"
+            - name: CRYOSTAT_AGENT_AUTHORIZATION
+              # Replace "abcd1234" with a base64-encoded authentication token
+              value: "Bearer abcd1234"
+              # This environment variable is key to the "hybrid" setup. This instructs the Agent to register itself with Cryostat as reachable via JMX, rather than reachable via HTTP.
```

Comment:
Could we break this long comment into 2 lines? Just nice to avoid long scrolling.